haas/api.py

```diff
@@ -822,6 +813,9 @@ def show_node(nodename):
         'free': node.project_id is None,
         'nics': [{'label': n.label,
                   'macaddr': n.mac_addr,
+                  'networks': dict([(attachment.channel,
```

Comment:
Looks like [json.dump()](https://docs.python.org/2.7/library/json.html#json.dump) supports this with `sort_keys=True`!